src/plugins/VideoRoomPlugin.ts

```diff
@@ -381,11 +394,17 @@ export class VideoRoomPlugin extends BasePlugin {
     const jsepOffer = await this.rtcConnection.createOffer(this.offerOptions);
     await this.rtcConnection.setLocalDescription(jsepOffer);
 
-    const confResult = await this.sendMessage({
-      request: 'configure',
-      ...options,
-    }, jsepOffer);
+    const confResult = await retryPromise(
+      () => this.sendMessage({
+        request: 'configure',
+        ...options,
```

Comment:
Maybe it's better to put options spread above request parameter assignment to avoid potential overriding

Reply:
This is how it's done by voicecenter. I prefer to change logic as little as possible for our needs to simplify possible merges in future